### DISCUSSION TITLE
Simple duplicate device to new one

### DISCUSSION THREAD

**Discussion by TomasAdvance** [frangoteam/FUXA#621] [General]
Hi Fuxa team,
is it possible to duplicate one device and create new one with same tags?
Thanks for your answer.
T

**Comment by unocelli** [comment]
Hi,
you can export the devices in json or svg file, edit it and import again, don't forget that the id are uuid.

**Comment by TomasAdvance** [comment]
Hi, I have tested by json. Honestly it is really very complicated.
It will be new future simple duplicate device. That will be very helpful if you have project where is same type of device whit same tag's.
Can you thinking about?

**Comment by unocelli** [comment]
Hi, we have not yet planned this extension, you can try with CSV export and modify it with Excel
